src/Layout/Navigation/__tests__/MainItem.spec.tsx

```diff
@@ -20,8 +20,12 @@ import { NavigationItemObject } from '..';
 import { FakeStoreBuilder } from '../../../store/__mocks__/storeBuilder';
 import { Provider } from 'react-redux';
 
+const _expect = expect as any;
```

Comment:
hello, I'm curious why we need to do that ?

Reply:
@benoitf  We need to do that because of wrong types. Somehow it resolved in a wrong way after migration.
I added that as a quick fix.

Reply:
ok thanks

Reply:
Hmm, I've just migrated the dashboard to yarn2 from scratch and all tests passed as is. No fixes are necessary.

Reply:
it's because I've put `nodeLinker: node-modules` into my `.yarnr.yml`, while `PnP` breaks tests.

Reply:
Hey, just cloned and checked, the type of `expect` is `jest.expect` so it works with PnP

Reply:
@merceyz Thanks a lot

Reply:
Fixed